<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindDialog</class>
 <widget class="QDialog" name="FindDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>115</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>425</width>
    <height>115</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>425</width>
    <height>115</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Find</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="2">
      <widget class="QRadioButton" name="uiDown">
       <property name="text">
        <string>&amp;Down</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="uiWholeWord">
       <property name="text">
        <string>Match whole &amp;word only</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QRadioButton" name="uiUp">
       <property name="text">
        <string>&amp;Up</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="uiMatchCase">
       <property name="text">
        <string>&amp;Match case</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Find:</string>
       </property>
       <property name="buddy">
        <cstring>uiFind</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="uiFind"/>
     </item>
     <item row="0" column="3" rowspan="3">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QPushButton" name="uiFindNext">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Find &amp;Next</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="uiClose">
         <property name="text">
          <string>&amp;Close</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>uiFind</tabstop>
  <tabstop>uiFindNext</tabstop>
  <tabstop>uiClose</tabstop>
  <tabstop>uiMatchCase</tabstop>
  <tabstop>uiWholeWord</tabstop>
  <tabstop>uiUp</tabstop>
  <tabstop>uiDown</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>uiClose</sender>
   <signal>clicked()</signal>
   <receiver>FindDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
